ceph.spec.in

```diff
@@ -1321,6 +1324,10 @@ env | sort
 mkdir -p %{_vpath_builddir}
 pushd %{_vpath_builddir}
 cmake .. \
+%if 0%{?suse_version} == 1500
```

Comment:
shouldn't this line also sync with above https://github.com/ceph/ceph/pull/48042/files#diff-34eff6689abb9572612fd8b57b58beefc7fa86fd2bb54efd27dfe628f67bfa62R206?
```
%if 0%{?suse_version} > 1500
```